Sources/BraintreePayPal/Public/BraintreePayPal/BTPayPalRequest.h

```diff
@@ -168,6 +168,11 @@ typedef NS_ENUM(NSInteger, BTPayPalRequestUserAction) {
  */
 @property (nonatomic) BOOL offerCredit;
 
+/**
+ Optional: Offers PayPal Pay Later if the customer qualifies. Defaults to false.
```

Comment:
Do we want to mention that this is only applicable to checkout flows (not billing agreement)?

Reply:
Or is it applicable to both but we only send the analytics event for checkout? Maybe I'm just confused by this code above:
```objc
if (paymentType == BTPayPalPaymentTypeCheckout && self.payPalRequest.offerPayLater) {
        NSString *eventName = [NSString stringWithFormat:@"ios.%@.webswitch.paylater.offered.%@", [self.class eventStringForPaymentType:paymentType], success ? @"started" : @"failed"];

        [self.apiClient sendAnalyticsEvent:eventName];
}
```